daily_spellbook/packages.yml

```diff
@@ -0,0 +1,7 @@
+packages:
+  - package: dbt-labs/dbt_utils
+    version: 1.1.1
+  - package: dbt-labs/codegen
+    version: 0.12.1
+  - package: starburstdata/trino_utils
+    version: 0.6.0
```

Comment:
these are new? Should we also add them to the main `packages.yml`?

Reply:
Yes probably, @jeff-dude added these files but we can update main in a separate PR

Reply:
these were added during @couralex6 discovery of subprojects during tokens build. i believe codegen was intended to help the process of moving files around, but not actively used? then trino_utils is just trino-specific dbt_utils. we can just add per subproject as needed. as of now, i'm not sure we actually use any from there

Reply:
maybe i'll remove from this new subproject, to ensure it doesn't break anything without